src/app/fyle/dashboard/dashboard.page.ts

```
@@ -117,6 +119,155 @@ export class DashboardPage {
+          const newConfig = {
+            ...featureConfigParams,
+            value: newConfigValue,
+          };
+
+          return this.featureConfigService.saveConfiguration(newConfig);
+        })
+      )
+      .subscribe(noop);
+  }
+
+  initializeTour(isApprover: Boolean): void {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)